.circleci/config.yml

```
@@ -72,7 +72,8 @@ jobs:
           from: packages/IconFont/fonts
           to: s3://welcome-ui/public/fonts/$BRANCH/$ICON_FONT_HASH/
           arguments: |
-            --metadata GitCommit=$CIRCLE_SHA1 --delete
+            --metadata GitCommit=$CIRCLE_SHA1 --delete \
+            --cache-control "max-age=2678400"
```

Que penses-tu @maxd-wttj de rajouter `immutable` donné qu'il y a un hash ?
```suggestion
            --cache-control "public, max-age=2678400, immutable"
```

---

C'est une bonne idée effectivement, je pense qu'on peut faire comme ça